internal/cephfs/volumeoptions.go

```diff
@@ -298,12 +298,10 @@ func newVolumeOptionsFromVolID(ctx context.Context, volID string, volOpt, secret
 	}
 
 	volOptions.ProvisionVolume = true
```

Comment:
there is no need for this PR to remove unneeded empty lines. The single commit is already large enough that reviewing is non-trivial.

Reply:
its needed for the linter to be happy. will place it in a seperate commit